packages/uhk-web/src/app/components/macro/action-editor/tab/mouse/macro-mouse.component.ts

```diff
@@ -7,6 +7,7 @@ import {
     MacroSubAction
 } from '../../../../../config-serializer/config-items/macro-action';
 import { Tab } from '../../../../popover/tab';
+import { MacroBaseComponent } from './../macro-base.component';
```

Comment:
pls remove ./ from beginning of the import

Reply:
thx.